bin/github-changelog.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+require __DIR__ . '/../vendor/erusev/parsedown/Parsedown.php';
+
+define( 'GITHUB_ENDPOINT', 'https://api.github.com/repos/Automattic/vip-go-mu-plugins/pulls?per_page=10&sort=updated&direction=desc&state=closed');
+define( 'WP_CHANGELOG_ENDPOINT', 'https://public-api.wordpress.com/wp/v2/sites/wpvipchangelog.wordpress.com/posts');
+define( 'WP_CHANGELOG_AUTH_TOKEN', $_SERVER['CHANGELOG_POST_TOKEN'] );
+define( 'PR_CHANGELOG_HEADER', '<h2>Changelog Description' );
+define( 'PR_DESCRIPTION_HEADER_PREFIX', '<h2>' );
+
+function fetch_last_PR() {
+    $ch = curl_init( GITHUB_ENDPOINT );
+    curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+    curl_setopt($ch, CURLOPT_HEADER, 0);
+    curl_setopt($ch, CURLOPT_HTTPHEADER, [ 'User-Agent: script' ] );
+    $data = curl_exec($ch);
+    curl_close($ch);
+
+    return json_decode($data, true)[0];
```

Comment:
I think this will work fine most of the times, but there could be some race conditions (e.g, someone posts a comment to another PR before the CircleCI job is executed). A safer approach would be to look at all fetched PRs for the `merged_at` field, and simply keep the PR with the most recent merge time.

Reply:
Great idea. Will do.

Reply:
done

Reply:
Hum... I think I didn't explain myself :sweat_smile:   I meant not just filtering by PRs with a `merged_at` field, but also traversing all of them and keeping the one with the most recent `merged_at` (i.e, the last merged PR, instead of the last updated one, that might be due to some comment or changed label)

Reply:
Of course in an extreme case it might happen that the most recently merged PR is not in the list of the 10 most recent updated ones, but I doubt that's a real possibility 😉

Reply:
Oh, I thought you were worried it would pick some other PR that was just closed without merging... in both cases it seems like really an edge case, right? Somebody would have to comment on one of the merged PRs between the time the last pr was merged and the build pipeline was finished.

I don't think that would happen that often ...and the impact is kinda small. But if you think it is worth doing I will check it out some more tomorrow.

Reply:
More than a comment I would expect some other change (an added/removed label, e.g). But yes, probably the case of a dismissed PR would be the most plausible, and that's covered with your recent change. As any other case would be rare, we can leave it this way, and see how it behaves